example/service.js

```diff
@@ -1,33 +1,19 @@
-let debug = require('debug')('example:service')
+let debug = require('debug')('service')
 let debugZmq = require('debug')('service:zmq')
 let debugZmqTx = require('debug')('service:zmq:tx')
-// let indexd = require('indexd') // XXX: you should use npm published package typically
-let indexd = require('../')
+let Indexd = require('../indexd') // FIXME: use require('indexd') in production
```

Comment:
This line change breaks the example.
I think it should be:
`let Indexd = require('../')`